WordPress/JetpackStatsWidgets/Model/HomeWidgetData.swift

```diff
@@ -6,9 +6,7 @@ import BuildSettingsKit
 extension HomeWidgetData {
 
     static func read(from cache: HomeWidgetCache<Self>? = nil) -> [Int: Self]? {
-
-        let cache = cache ?? HomeWidgetCache<Self>(fileName: Self.filename,
-                                                                  appGroup: BuildSettings.appGroupName)
+        let cache = cache ?? makeCache()
```

Comment:
Duplicated this for simplicity. It's just convenience wrappers – I don't think it hurts. It's better than creating yet another module just to reuse two methods.